test/phpunit/phpunittest.xml

```diff
@@ -1,45 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE phpunit SYSTEM "phpunit.dtd" >
 <phpunit
-	backupGlobals="false"
-	backupStaticAttributes="true"
-	convertErrorsToExceptions="true"
-	convertNoticesToExceptions="true"
-	convertWarningsToExceptions="true"
-	colors="true"
-	processIsolation="false"
-	stopOnFailure="true">
-	<php>
-		<ini name="max_execution_time" value="1800"/>
-		<ini name="max_input_time" value="1800"/>
-        <ini name="memory_limit" value="4G" />
-        <!-- Also don't forget to change suhosin.memory_limit=4G -->
-	</php>
-	<filter>
-	  <whitelist addUncoveredFilesFromWhitelist="true">
-	    <directory suffix=".class.php">../../htdocs/</directory>
-	    <directory suffix=".lib.php">../../htdocs/</directory>
-	    <directory suffix=".inc.php">../../htdocs/</directory>
-	    <directory suffix=".modules.php">../../htdocs/</directory>
-	    <file>../../htdocs/core/modules/facture/modules_facture.php</file>
-	    <file>../../htdocs/core/modules/commande/modules_commande.php</file>
-	    <file>../../htdocs/core/modules/propale/modules_propale.php</file>
-	    <file>../../htdocs/core/modules/project/modules_project.php</file>
-	    <file>../../htdocs/core/modules/fichinter/modules_fichinter.php</file>
-	    <exclude>
-	    <directory suffix=".php">../../build/</directory>
-	    <directory suffix=".php">../../dev/</directory>
-	    <directory suffix=".php">../../doc/</directory>
-	    <directory suffix=".php">../../test/</directory>
-	    <directory suffix=".php">../../documents/</directory>
-	    <directory suffix=".php">../../htdocs/custom/</directory>
-	    <directory suffix=".php">../../htdocs/documents/custom/</directory>
-	    <directory suffix=".php">../../htdocs/nltechno/</directory>
-	    <directory suffix=".php">../../htdocs/products/canvas/</directory>
-	    <directory suffix=".php">../../htdocs/contact/canvas/</directory>
-	    <directory suffix=".php">../../htdocs/societe/canvas/</directory>
-	    <directory suffix=".php">../../htdocs/includes/</directory>
-	    </exclude>
-	  </whitelist>
-	</filter>
+  backupGlobals="false"
+  backupStaticAttributes="true"
+  convertErrorsToExceptions="true"
+  convertNoticesToExceptions="true"
+  convertWarningsToExceptions="true"
+  colors="true"
+  processIsolation="false"
+  stopOnFailure="false">
```

Comment:
As we have to pay for CPU used by travis, we don't want to continue if an error is found.
Knowing there is an error is enough to stop consumption.